libdirections-offboard/src/main/java/com/mapbox/navigation/route/offboard/router/MapboxDirectionsUtils.kt

```diff
@@ -0,0 +1,226 @@
+package com.mapbox.navigation.route.offboard.router
```

Comment:
Can we move this class to base and a add `@file:JvmName("MapboxDirectionsUtils")`?

Reply:
there is an extension `MapboxDirections.Builder.routeOptions` . Base has no dependency on `MapboxDirections`. Right?

Reply:
Yup, good catch. Let's move that one extension to a different file and leave it in offbaord module and move the rest to base.

Reply:
done